golang/cosmos/x/vtransfer/ibc_middleware_test.go

```diff
@@ -540,6 +550,8 @@ func (s *IntegrationTestSuite) TestHops() {
 
 					s.coordinator.CommitBlock(nextPath.EndpointA.Chain, nextPath.EndpointB.Chain)
 
+					denomTrace = s.prependDenomTrace(nextPath.EndpointB, denomTrace)
```

Comment:
Nice this replicates the wrapping of tokens done in IBC transfers, right?

Reply:
Precisely!